config.py

```diff
@@ -40,7 +40,10 @@ def setup(self, cfg):
 
 logger_class = CustomGunicornLogger
 
-# preload_app = True
+# Must be set to True, otherwise scheduler and messaging thread are started
+# multiple times in each of gunicorn worker.
+preload_app = True
```

Comment:
```suggestion
preload_app = FLASK_ENV != 'development'
```

> The reloader is incompatible with application preloading.

https://docs.gunicorn.org/en/stable/settings.html#reload

I'm not sure if you tried this, but if the incompatibility applies in this case, we may need to remove the 'reload' configuration.

Reply:
Fixed, I applied your suggestion.

It is a bit tricky, from logs it looks like reloading works, but it doesn't.